$MirOS: ports/lang/nwcc/patches/patch-cpp_cpp_main_c,v 1.1 2008/10/30 17:11:16 tg Exp $
--- cpp/cpp_main.c.orig	Wed Mar  4 16:13:06 2009
+++ cpp/cpp_main.c	Fri Apr 17 14:24:23 2009
@@ -229,6 +229,11 @@ static struct input_file	nullinput;
 		sprintf(buf, "%d", __OpenBSD__);
 		tmp_pre = tmp_pre_macro("__OpenBSD__", buf);
 		APPEND_LIST(predef, predef_tail, tmp_pre);
+#if defined(__MirBSD__)
+		sprintf(buf, "%d", __MirBSD__);
+		tmp_pre = tmp_pre_macro("__MirBSD__", buf);
+		APPEND_LIST(predef, predef_tail, tmp_pre);
+#endif
 #elif defined linux 
 		sprintf(buf, "%d", linux);
 		tmp_pre = tmp_pre_macro("linux", buf);
